CONTRIBUTING.md

```diff
@@ -39,7 +41,7 @@ The name of the branch is not really relevant. Nonetheless we usually go for a p
 Theres a nifty script that creates that files you'll need:
 
 ```
```

Comment:
```suggestion
```shell
```
And same for the other code blocks

Reply:
Will leave these two blocks without the `shell` language set because they are not shell scripts:

```
<type>[optional scope]: <description>
```

And 

```
feat: Add CoolCatsAndKittens Component
```